### PR TITLE
Fix timeout

### PR DESCRIPTION
**Pull request by unblee**
## RELATED ISSUES

#3 

## WHY

Function stops due to operation for a long time(30min~).

## WHAT

When a timeout occurs, create a watch object again.

### CODE REVIEW

File: client.go [hunk 1]
```diff
@@ -52,190 +52,184 @@ func newClient(origin, ignoreDestCSV string) (*client, error) {
...
+					ObjectMeta: metav1.ObjectMeta{
+						Name:      secret.ObjectMeta.Name,
+						Namespace: newNS,
+					},
+					Type: secret.Type,
+					Data: secret.Data,
+				})
 				if err != nil {
-					log.Printf("failed to receive a secret list from '"+c.origin+"': %s\n", err)
+					log.Printf("failed to copy a secret '"+secret.ObjectMeta.Name+"' from '"+c.origin+"' to "+newNS+": %s\n", err)
```
Comment (dtan4) [inline client.go]: Please use `%s` instead of String concatenation.

**Comment by dtan4** [comment]
LGTM 👍

File: client.go (hunk 1 above)
Comment (unblee) [reply client.go]: Fixed!

File: main.go [hunk 2]
```diff
@@ -61,7 +61,7 @@ func main() {
 		for {
 			stop, err := client.watchSecretEvents(ctx, wg)
 			if err != nil {
-				log.Printf("failed to start watching of secret events: %s\n", err)
```
Comment (dtan4) [inline main.go]: good catch 👍

**Comment by dtan4** [comment]
LGTM 👍

**Comment by unblee** [comment]
Thanks!

**Pull request merged**